plugins/available-plugins.json

```
@@ -47,5 +47,12 @@
     "description": "Custom style for Baidu Health med-style",
     "repo": "https://github.com/KangXinzhi/tempad-dev-plugin-med-style",
     "url": "https://raw.githubusercontent.com/KangXinzhi/tempad-dev-plugin-med-style/refs/heads/main/dist/index.mjs"
-  }
+  },
+  {
+    "name": "tailwind",
+    "author": "@haydenull",
+    "description": "A plugin for tempad-dev that transforms CSS to Tailwind CSS",
```

```suggestion
    "description": "CSS to Tailwind CSS",
```

Let's make it shorter.